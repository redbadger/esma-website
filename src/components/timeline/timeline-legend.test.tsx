import React from "react";
import renderer from "react-test-renderer";
import TimelineLegend from './timeline-legend';
import SchoolYearsIcon from "../../svg/pencil.svg";


describe("TimelineLegend", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<TimelineLegend entries={entries}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});


const entries = [
  {
    title: "test",
    imageName: "test.jpg",
    description: "Test",
    icon: SchoolYearsIcon,
    destination: "/issues-chronology/early-years/overview",
    color: "aqua",
    anchor: "early-years",
  }]
  