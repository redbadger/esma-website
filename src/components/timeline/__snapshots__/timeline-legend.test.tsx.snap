// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimelineLegend renders correctly 1`] = `
<div
  className="css-19s79zm-containerStyles"
>
  <div
    className="css-1srnxw3-timelineLegendStyles"
  >
    <a
      href="#early-years"
    >
      <div
        className="css-15zjo8g-entryWrapperStyles-TimelineLegend"
      >
        <test-file-stub />
        <span>
          test
        </span>
      </div>
    </a>
  </div>
</div>
`;
